sisl/viz/plotly/_user_customs.py

```diff
@@ -0,0 +1,130 @@
+import os
+from pathlib import Path
+import sys
+import importlib
+
+from .._env_vars import register_env_var
+
+__all__ = ["import_user_presets", "import_user_plots",
+    "import_user_sessions", "import_user_plugins"]
+
+# Define the folder where the user will store their stuff
+DEFAULT_USER_CUSTOM_FOLDER = Path.home() / ".sisl"
+USER_CUSTOM_FOLDER = register_env_var(
+    'USER_DIR', DEFAULT_USER_CUSTOM_FOLDER,
```

Comment:
I don't know if we will later add more stuff. But perhaps it would make sense to have a directory structure that follows sisl.

I.e. `plotly` templates go in `$HOME/.sisl/viz/plotly`?

This would be handy for blender as well `$HOME/.sisl/viz/blender`

Reply:
Yes, it is probably the best way of doing it, I will change it.